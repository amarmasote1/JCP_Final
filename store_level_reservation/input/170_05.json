[
  {
    "storeID": "2044",
    "itemID": "15345672209",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 1
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2047",
    "itemID": "15345672209",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 1
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2048",
    "itemID": "15345672209",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 10
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2044",
    "itemID": "15345672210",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2047",
    "itemID": "15345672210",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2048",
    "itemID": "15345672210",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]