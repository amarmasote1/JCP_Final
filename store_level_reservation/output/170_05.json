{
    "message": "Success",
    "body": [
        {
            "itemID": "15345672209",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2044",
            "supplyData": {
                "onhandsupply": 1
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672209",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2047",
            "supplyData": {
                "onhandsupply": 1
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672209",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2048",
            "supplyData": {
                "onhandsupply": 10
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672210",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2044",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672210",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2047",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672210",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2048",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        }
    ],
    "status": 200
}