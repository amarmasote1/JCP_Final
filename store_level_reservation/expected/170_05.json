{"status":200,"message":"Success","body":[{"itemID":"15345672209","uom":"EACH","storeID":"2044","supplyData":{"onhandsupply":1},"demandData":{"reserveddemand":0}},{"itemID":"15345672209","uom":"EACH","storeID":"2047","supplyData":{"onhandsupply":1},"demandData":{"reserveddemand":0}},{"itemID":"15345672209","uom":"EACH","storeID":"2048","supplyData":{"onhandsupply":10},"demandData":{"reserveddemand":0}},{"itemID":"15345672210","uom":"EACH","storeID":"2044","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672210","uom":"EACH","storeID":"2047","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672210","uom":"EACH","storeID":"2048","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}}]}